### PR TITLE
Fix for #668, call_required_libraries() is badly implemented (not self-documenting).

### PR DESCRIPTION
**Pull request by Trodrige**
Adjusted call_required_libraries() function's parameters and arguments to be explicit.

### CODE REVIEW

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'libreehr1';
```
Comment (nileshprasad137) [inline sites/default/sqlconf.php]: This file should not be touched. Please use gitignore.

Comment (aethelwulffe) [reply sites/default/sqlconf.php]: What he said...

File: templates/practice_settings/general_list.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <html>
 <head>
 <link rel="stylesheet" href="<?php echo $this->css_header;?>" type="text/css">
-<?php call_required_libraries(true,true,false,false);?>
+<?php call_required_libraries(bootstrap=true,fancybox=true,knockout=false,datepicker=false); ?>
```
Comment (aethelwulffe) [inline templates/practice_settings/general_list.php]: I guess we are still using the /templates directory?

Comment (Trodrige) [reply templates/practice_settings/general_list.php]: ok,  I'm looking into it @aethelwulffe

Comment (nileshprasad137) [reply templates/practice_settings/general_list.php]: Yes, but the smarty templates are removed.

**Comment by aethelwulffe** [comment]
This meets with Nilesh's requirements, and Terry approves.  I am merging.

**Comment by Trodrige** [comment]
@aethelwulffe concerning this "look in the docs (after you have fixed the config file issue in this commit) and make sure the function gets documented." 
Please more light on documenting the function, I don't get it clearly.

**Comment by aethelwulffe** [comment]
@Trodrige  I was just talking about looking through the /docs directory for developer documentation (api) related to this include function, then creating it or updating it to list all the implemented named calls (and order of the same, as I think this is still an ordered list of arguments).  I would dig through for you, or will update it myself later if you like.  I was just putting this out there 'cause I am in a nest of spaghetti code at the moment, and trying to dig myself out with my tiny, tiny dried up brain.

**Pull request merged**